### ISSUE TITLE
Commit uncompiled version

### ISSUE THREAD

**Issue by iansilber** [abynim/SketchSquares#16]
Have you considered adding an uncompiled version of the .app portion for people who might want to peek in to that portion?

**Comment by abynim** [comment]
Yes, but it is quite messy at this point. It was written in Swift and needs to be updated for the latest version of Swift. I'll clean it up one of these weekends and make it available as a separate repo.
